#include "next_level_exception.h"

const char* next_level_exception::what() const
{
    return "Going to the next level!";
}
